docs/getting-started.md

```diff
@@ -139,6 +139,17 @@ Now that you're setup, if you can run it on the command line, you can run it in
     4. For `Bazel Command` you can put any bazel command, like `build`, `run`, `test`, etc.
     5. Click `Ok`, then there should be a green arrow in the top right corner by the drop-down menu. Click it and the test will run!
 
+### Running our AI, Simulator or Robot Diagnostics
+
+1. Open your terminal and run `ifconfig`.
+2. Pick the network interface you would like to use:
+    1. If you are running things locally, you can pick any interface that is not `lo`
+    2. If you would like to communicate with robots on the network, make sure to select the interface that is connected to the same network as the robots.
+3. Run AI: `bazel run //software:full_system -- --interface=interface_here --backend=WifiBackend`
```

Comment:
nit:
```suggestion
3. Run AI: `bazel run //software:full_system -- --interface=[interface_here] --backend=WifiBackend`
```

and same for below, but it's a personal preference thing, so your call

Reply:
fixed